docs/sql/sql.md

```diff
@@ -0,0 +1,440 @@
+# SQL Support
+
+SpacetimeDB supports two subsets of SQL.
+One for ad hoc queries against the database.
+These can be made through the [cli] or the [http] endpoint.
+Another for subscriptions made through the [websocket] api or [sdk].
+
+## Subscriptions
+
+```ebnf
+SELECT projection FROM relation [ WHERE predicate ]
+```
+
+The subscription language is a query language.
+It does not support data manipulation commands.
+It is used to subscribe to a set of rows from a single table.
```

Comment:
Below it says we can use joins, so sounds contradictory.

Reply:
Is it made clear as you read on, or do you think more clarification is needed?

Reply:
Maybe is more clear if says:

The subscription language is strictly a query language designed specifically for subscribing to a set of rows from a single table, so is not allowed to do data manipulation commands.